resources/default.nms

```diff
@@ -395,11 +389,11 @@
 		Double getDirectionX()
 		Double getDirectionY()
 		Double getDirectionZ()
-		# Gets the entity's current velocity in the x direction.
+		# Gets the entity's current acceleration (yes, acceleration) in the x direction.
```

Comment:
After some testing and discussion in discord, it became clear that this function indeed returns velocity, not acceleration, but has problems handling players (works normally on other entities). I propose changing the text to something like `Gets the entity's current velocity in the x direction. Works unreliably if the entity is a player`.

Reply:
@Chance103 I cannot merge this pull request, because the proposed function description isn't correct (the function returns velocity, not acceleration, but just behaves unexpectedly because velocity is client side, and server has to guess it). I'm open to discussing the exact wording if you want, but please update the pull request with requested changes, or I'll have to close it and commit changes separately.